exec/cnex/exec.h

```diff
@@ -40,6 +49,15 @@ typedef struct tagTExecutor {
         size_t total_allocations;
         size_t collected_objects;
     } diagnostics;
+
+    BOOL debugging;
+    HttpServerImpl *debug_server;
+    IHttpServerHandler *handlers;
+    DebuggerState debugger_state;
+    int64_t debugger_step_source_depth;
+    char *debugger_breakpoints;
+    int num_breakpoints;
```

Comment:
Removed.  That variable may come back if I decide to use a linked-list, or an array of breakpoints, rather than using a char to indicate bytecode addresses.